app/views/microposts/_form.html.erb

```diff
@@ -6,17 +6,17 @@
       <div>
         <div>
 
-          <%= form.label :content %>
-          <%= form.text_area :content %>
+              <%= form.label :content %>
+              <%= form.text_area :content %>
+              <%= form.date_field :published_at, as: :string, required: true, data: { controller: "flatpickr", flatpickr_min_date: Time.zone.now #disables past dates
```

Comment:
@HrisikeshSinghaChowdhury Where is the migration for adding published_at column for blog

Reply:
what is the user of data: { controller: "flatpickr", flatpickr_min_date: Time.zone.now } ? 
Remove this options and try calling flatpickr using css class  selector